packages/astro/src/core/app/node.ts

```diff
@@ -34,6 +34,24 @@ export class NodeApp extends App {
 		return super.match(req instanceof Request ? req : createRequestFromNodeRequest(req), opts);
 	}
 	render(req: IncomingMessage | Request, routeData?: RouteData) {
+		// @ts-ignore
```

Comment:
why is this ts-ignore? If it's the types of `req.body` I'd prefer if you cas to `any`